src/components/app.js

```diff
@@ -78,13 +88,19 @@ const App = () => {
       });
   };
   const handleChangePlan = (p) => {
+    if (p === "custom") {
```

Comment:
The logic may conflict with line 55-65, I think we should change that part to cover this case.